src/overmap.cpp

```diff
@@ -2370,9 +2370,10 @@ void overmap::move_hordes()
             if(add_to_group == NULL) {
                 mongroup m(GROUP_ZOMBIE, p.x, p.y, p.z, 0, 1);
```

Comment:
> 0 currently signifies that the group can be deleted, but I could change that check to also check the list for emptiness.
> It only uses the population if the monsters list is empty. Yeah, I can see why that's confusing, but not sure how to fix without subclassing or something.

Add an `empty` function to `mongroup` that checks for both things being empty (the monster list and the population), use that function instead of comparing population to 0 for the ready-to-be-deleted check.